serialized-lob/src/main/java/com/iluwatar/serialized/lob/App.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * The MIT License
+ * Copyright © 2014-2021 Ilkka Seppälä
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package com.iluwatar.serialized.lob;
+
+import java.io.IOException;
+import java.sql.SQLException;
+import javax.sql.DataSource;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import lombok.extern.slf4j.Slf4j;
+import org.h2.jdbcx.JdbcDataSource;
+
+import java.util.List;
+
+/**
+ * App.
+ */
+@Slf4j
+public final class App {
+  private static final String DB_URL = "jdbc:h2:~/test";
+
+  /**
+   * Private Constructor.
+   *
+   */
+  private App() {}
+
+  /**
+   * Program entry point.
+   *
+   * @param args command line args.
+   * @throws SQLException if any error occurs.
+   */
+  public static void main(String[] args) throws SQLException, IOException, ClassNotFoundException {
+    // Create data source and create the person table.
+    final var dataSource = createDataSource();
+
+    // Drop table Positions if exists
+    deleteSchema(dataSource);
+
+    // Create table Positions if not exists
+    createSchema(dataSource);
+
+    // Initialize five departments for different positions.
+    var devDept = new Department("Developing", null);
+    var testDept = new Department("Testing", null);
+    var devManageDept = new Department("Development Management", List.of(devDept));
+    var testManageDept = new Department("Testing Management", List.of(testDept));
+    var engineeringManageDept = new Department("Engineering Management", List.of(devManageDept, testManageDept));
```

Comment:
Although there's nothing wrong with the topic of this example, I think 99% of the readers would prefer something more interesting like dragons, trolls, space rockets, or black holes.

Reply:
I totally get it, it would definitely be more fun! Unfortunately, I am kind of working on a tight schedule with many things and changing the topic of example might impact many of the parts...Also we, as a team, are working on these issues towards merging as many as possible before this Friday (12/10) for the term project, so I was wondering if there is any chance we could maybe proceed with this rather boring topic? I would happy to change the example to be around black holes if I had more time...